### DISCUSSION TITLE
Gremlin Endpoint connection using managed identity

### DISCUSSION THREAD

**Discussion by jayaemekar** [Gremlinq/ExRam.Gremlinq#1221] [General]
I wanted to check, does cosmos no-SQL for Apache gremlin provide any support for managed identity auth? , If its not supported can you help me with related documentation?

**Comment by danielcweber** [comment]
I am not aware that CosmosDb is accessible through a managed identity, but only through an AuthKey. An Azure KeyVault, however, works 
 well with managed identities AFAIK, so storing and retrieving the AuthKey from there is maybe an option.
